apps/sms/test/marionette/conversion_banner_test.js

```diff
@@ -0,0 +1,114 @@
+
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+var assert = require('chai').assert;
+
+var Messages = require('./lib/messages.js');
+
+marionette('Message Type Conversion Banner', function() {
+  var apps = {};
+
+  var client = marionette.client({
+    prefs: {
+      'focusmanager.testmode': true
+    },
+    settings: {
+      'lockscreen.enabled': false,
+      'ftu.manifestURL': null
+    },
+
+    apps: apps
+  });
+
+  var messagesApp;
+
+  function assertIsDisplayed(element) {
+    assert.isTrue(element.displayed(), 'Element should be displayed');
+  }
+
+  function assertIsNotDisplayed(element) {
+    assert.isFalse(element.displayed(), 'Element should not be displayed');
+  }
+
+  function waitForBannerToDisappear() {
+    client.helper
+      .waitForElementToDisappear(messagesApp.Composer.conversionBanner);
+  }
+
+  setup(function() {
+    messagesApp = Messages.create(client);
+
+    client.contentScript.inject(
+      __dirname + '/mocks/mock_navigator_moz_icc_manager.js'
+    );
+    client.contentScript.inject(
+      __dirname + '/mocks/mock_navigator_moz_mobile_message.js'
+    );
+  });
+
+  suite('Message Type Conversion Banner Test Suite', function() {
+
+    setup(function() {
+      messagesApp.launch();
```

Comment:
Could you please create  simple sms thread in setup and in the tests below (probably as the last case for every test) navigate to it and verify that no banner is shown. I remember we had such issue during one of the past reviews (go to mms-email-thread - > mms banner shown -> go to thread list -> go to ordinary thread -> sms banner is shown ---- wrong) so it would be safe to check that we haven't broken old behavior.